types/swagger-ui-dist/index.d.ts

```diff
@@ -20,15 +20,19 @@ interface Url {
 
 interface SwaggerUIBundle {
   (a: {
-    deepLinking: boolean;
-    dom_id: string;
-    presets: any[];
-    plugins: any;
-    urls: Url[];
-    layout: string;
+    configUrl?: string;
+    deepLinking?: boolean;
+    dom_id?: string;
+    domNode?: string;
+    presets?: any[];
+    plugins?: any;
+    spec?: any;
+    url?: Url;
```

Comment:
type `url` should be a `string` by documentation

url | String. The URL pointing to API definition (normally swagger.json or swagger.yaml). Will be ignored if urls or spec is used.
-- | --

Reply:
Thanks great catch 👍 Fixed here: https://github.com/DefinitelyTyped/DefinitelyTyped/pull/37115/commits/f2ea4bfc2dbbb46ed784255eb961d3ecf1f416b8